frontend/app/components/modal/ResetPasswordModal.vue

```diff
@@ -0,0 +1,57 @@
+<script setup lang="ts">
+import type { FormSubmitEvent } from '@nuxt/ui'
+import * as z from 'zod'
+
+const schema = z.object({
+  code: z.string({ error: 'Kod jest wymagany' }),
+  password: z.string({ error: 'Hasło jest wymagane' }).min(8, 'Hasło musi mieć minimum 8 znaków'),
```

Comment:
This is copied from other place, i think we can create global definition for it. I can look into that in a few hours